src/GeoDB.cpp

```diff
@@ -42,7 +42,7 @@ MaxmindDB::~MaxmindDB()
     }
 }
 
-std::string MaxmindDB::getGeoLocString(const struct sockaddr *sa) const
+std::string MaxmindDB::getGeoLocString(const struct sockaddr *sa, bool &error) const
```

Comment:
instead of calling it `error`, should we call it `notfound`? it's not an actual error but rather a status/condition